docs/user-guide/running-sipnet.md

```diff
@@ -0,0 +1,346 @@
+# Running SIPNET
+
+This guide covers how to run SIPNET with command-line options and configuration files.
+
+## Quick Start
+
+The basic command to run SIPNET is:
+
+```bash
+./sipnet
+```
+
+This will look for an input configuration file named `sipnet.in` in the current directory. If found, it reads settings from that file. If not found, it uses built-in defaults.
+
+To use a different configuration file:
+
+```bash
+./sipnet --input-file myconfig.in
+```
+
+To use different input file prefixes (climate and parameter files):
+
+```bash
+./sipnet --file-name mysite
+```
+
+This will look for `mysite.clim` and `mysite.param` files.
+
+## Command-Line Options
```

Comment:
Explain that these options can be set either in sipnet.in or at command line. Then I think you could combine the two sections and just say that these can be set in one or the other. (and I think cli has precedence but please confirm)